owasp-top10-2017-apps/a2/saidajaula-monster/app/app.py

```diff
@@ -8,10 +8,13 @@
 import json
 import hashlib
 import uuid
+import hmac
 from functools import wraps
 
 
 app = Flask(__name__)
+app.config['KEY'] = os.urandom(64).encode('hex')
```

Comment:
This approach generates unique keys for each process where applications are running. This behavior can be unacceptable from the user perspective because if the user connects at another server, your cookie HMACs will be invalid.
You can use environment variables to get a shared key.